MatomoTracker/MatomoTracker.swift

```diff
@@ -399,6 +399,25 @@ extension MatomoTracker {
             dimension.index != index
         })
     }
+    
+    /// remove all without ignore
+    @objc public func removeAll(ignoreIndexs indexs: [NSNumber]? = nil) {
```

Comment:
I think methods only should expose Swift types since the framework is written in Swift.
Any bridging from Objc types is the callers responsibility.

````
func removeAll(ignoreIndexes indexes: Set<Integer>? = nil)
````